src/AppTheme.js

```diff
@@ -0,0 +1,33 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+
+const theme = createMuiTheme({
+    palette: {
+        primary: {
+            main: '#C03736'
+        },
+        secondary: {
+            main: '#4F1315'
+        },
+        tertiary: {
+            main: '#C1D2D6'
+        }
+    },
+    status: {
+        danger: 'orange',
+    },
+    typography: {
+        fontFamily: [
+            'Poppins',
+            'Roboto',
+            'sans-serif',
+        ].join(','),
+        useNextVariants: true,
+    },
+
+}); 
+export default theme;
```

Comment:
Are we using a `ThemeProvider`? Might be better to not have to explicitly include the theme everywhere, just wrap some bottom level component, near `App` or something.

Reply:
We are doing that

Reply:
Yeah I found that out in the end, thank the file ordering 🙃  🤦‍♂ 

Ignore this and other comments relative to this, just resolve them straight away